2-mandatory/1-writers.js

```diff
@@ -70,8 +70,14 @@ Exercise 2:
   "Writer {firstName} {lastName} died at {age} years old."
 */
 
-function logDeadWritersInTheirForties() {
-  // write your code here
+function logDeadWritersInTheirForties( writers ) {
```

Comment:
You do not need to pass the array of writers into the functions. The scope of the variable is global, so it can be accessed within each function